test/e2e/quick_start_test.go

```diff
@@ -275,4 +277,40 @@ func TestQuickStart(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to ensure guest nodes became ready: %s", err)
 	}
+
+	clusterOperators := &configv1.ClusterOperatorList{}
+	err = wait.Poll(10*time.Second, 10*time.Minute, func() (done bool, err error) {
+		err = guestClient.List(ctx, clusterOperators)
+		if err != nil {
+			t.Logf("failed to list cluster operators: %s", err)
+			return false, nil
+		}
+		if len(clusterOperators.Items) == 0 {
+			return false, nil
+		}
+		ready := true
+		for _, clusterOperator := range clusterOperators.Items {
+			available := false
+			degraded := true
+			for _, cond := range clusterOperator.Status.Conditions {
+				if cond.Type == configv1.OperatorAvailable && cond.Status == configv1.ConditionTrue {
+					available = true
+				}
+				if cond.Type == configv1.OperatorDegraded && cond.Status == configv1.ConditionFalse {
+					degraded = false
+				}
+			}
+			if !available || degraded {
+				ready = false
```

Comment:
break here

Reply:
yeah, I left this when I thought I would log each CO that was not ready.